pkg/crc/gpg/gpg.go

```diff
@@ -0,0 +1,42 @@
+package gpg
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/ProtonMail/go-crypto/openpgp"
+	"github.com/code-ready/crc/pkg/crc/constants"
+)
+
+func Verify(filePath, signatureFilePath string) (err error) {
+	signed, err := os.Open(filePath)
```

Comment:
Might be just me, but `signed` feels a bit unspecific. `signedData`, `signedFile`, ... ? Or even omit the `signed` part of it?